src/server/journal/journal_slice.h

```diff
@@ -57,8 +57,10 @@ class JournalSlice {
   // with allow_flush=false and the subsequent call with allow_flush=true.
   void SetFlushMode(bool allow_flush);
 
+  size_t GetRingBufferSize() const;
```

Comment:
@adiholden do we want aggregate stats or per proactor ?

Reply:
yes and this is not only the ring buffer size, we should calculate the journal items size in the ring buffer as they contain strings which might be big